.travis.yml

```diff
@@ -9,13 +9,18 @@ jdk:
 stages:
   - test
   - build
+  - name: docker
+    if: branch = master && type != pull_request
 
 jobs:
   include:
     - stage: test
       script: ./gradlew test
     - stage: build
       script: ./gradlew packaging
+    - stage: docker
+      install: true
```

Comment:
is this the same as `install: skip` ?

Reply:
yes, it is the same
I think skip is newer and more descriptive

Reply:
We should add `install: skip` to all stages

Reply:
added it to all stages